### PR TITLE
Feature/display cover iteration 0 - begin functions for view covers

### PR DESCRIPTION
**Pull request by saraho1123**
## What is the change? 
We started query selectors and functions to display random covers on the home view of the web page.

## What does it fix? 
Nothing yet! We are just getting started!

## Is this a fix or a feature?  
Feature

## Where should the reviewer start? 
Line 8 of main.js

## How should this be tested?
This cannot be tested yet, as we do not yet have it functional!

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -17,3 +17,24 @@ var currentCover;
...
 }
+
+function displayCover() {
+  getRandomCovers();
+  // see a cover with a randomly selected image and taglin with 2 random discriptors
+  // loop through array of cover images using getRandomIndex(covers) function
+  // repeat that for title and 2 discriptors
+}
+
+function getRandomCovers() {
```
Comment (ADavidson02) [inline src/main.js]: We can refactor into one function and delete excess functions.

File: src/main.js [hunk 2]
```diff
@@ -19,22 +25,16 @@ function getRandomIndex(array) {
...
 function displayCover() {
-  getRandomCovers();
-  // see a cover with a randomly selected image and taglin with 2 random discriptors
-  // loop through array of cover images using getRandomIndex(covers) function
-  // repeat that for title and 2 discriptors
-}
-
-function getRandomCovers() {
-  getRandomIndex(covers);
+  coverImage.src = currentCover.cover;
```
Comment (ADavidson02) [inline src/main.js]: We need to look into why its not changing.

**Review by ADavidson02 — CHANGES_REQUESTED** [review]

File: src/main.js (hunk 2 above)
Comment (saraho1123) [reply src/main.js]: Thanks Amanda! Glad we were able to take a look at it and figure out what is happening.

**Comment by saraho1123** [comment]
As we are still learning how to properly do pull requests, we decided to go ahead and merge this to main, even though it is not correct. Then Amanda will get to start the next PR to merge the completed feature.

**Pull request merged**